integration_tests/testsuite/resources/gke.yaml.in

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```

Comment:
Are both this file and template.py requiered?

Reply:
nope, good call